src/main/client/config/client-config.ts

```diff
@@ -1,8 +1,22 @@
 import { CommonClientConfig } from './common-client-config';
 
+interface ServerConfig {
+   readonly url: string;
+   readonly name: string;
+}
+
 // Import this class for config properties, though, you should define here only development properties
 export class ClientConfig extends CommonClientConfig {
    static readonly ASSET_FOLDER: string = '../../src/asset/';
-   static readonly AUTO_START: boolean = true; // Skip main menu for development purposes
-   static readonly SERVER_HOST: string = `http://localhost:${ClientConfig.SERVER_PORT}`;
+   static readonly AUTO_START: boolean = false; // Skip main menu for development purposes
+   static readonly SERVER_HOSTS: ServerConfig[] = [
```

Comment:
There is already a ServerConfig on the server. We should use a name something like GameServer

Reply:
ok will change

Reply:
Or ServerHost as the variable name suggests

Reply:
Or rather ServerInstance